src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -110,6 +111,8 @@ function mapStateToProps() {
     stores: sortedStoresSelector(state),
     statOrder: state.dimApi.settings.loStatSortOrder.map((hash) => statHashToType[hash]),
     assumeMasterwork: state.dimApi.settings.loAssumeMasterwork,
+    minimumPower: state.dimApi.settings.loMinPower,
+    minimumStatTotal: state.dimApi.settings.loMinStatTotal,
```

Comment:
`settingsSelector(state).loMinStatTotal`

Though it's even easier to assign `settings` to a local once and then read stuff off of it.

Reply:
Yeah ok, I normally like to keep stuff scoped down to only what is needed but I can do that as there is a few of them.